Features/Intel/OutOfBandManagement/IpmiFeaturePkg/Include/PostMemory.fdf

```diff
@@ -11,13 +11,19 @@ INF IpmiFeaturePkg/GenericIpmi/Dxe/GenericIpmi.inf
 INF IpmiFeaturePkg/IpmiInit/DxeIpmiInit.inf
 INF RuleOverride = DRIVER_ACPITABLE IpmiFeaturePkg/BmcAcpi/BmcAcpi.inf
 INF IpmiFeaturePkg/BmcAcpiState/BmcAcpiState.inf
-INF IpmiFeaturePkg/BmcAcpiSwChild/BmcAcpiSwChild.inf
 INF IpmiFeaturePkg/BmcElog/DxeBmcElog.inf
-INF IpmiFeaturePkg/BmcElog/SmmBmcElog.inf
+INF IpmiFeaturePkg/GenericElog/Dxe/GenericElog.inf
+!if gMinPlatformPkgTokenSpaceGuid.PcdStandaloneMmEnable == TRUE

```

Comment:
One thought I had here... does FvAdvanced get processed by the Standalone MM dispatcher?

If no, then we should remove the standalone MM version of these drivers and board code will have to deal with it.